File: Logitude/Logitude.Test.Base/Constants/URLs.cs
namespace Logitude.Test.Base.Constants
{
    public static class URLs
    {
        //public static string UserAuthentication = "Authentication";
        //public static string TenantsGetSingle = "Tenants/GetSingle?id=";
        //public static string UserViewsGetByFilters = "UserViews/GetByFilters";
        //public static string ContactViewsGetByFilters = "ContactViews/GetByFilters";

        public static string UserAuthentication()
        {
            return "Authentication";
        }

        public static string TenantsGetSingle(int id)
        {
            return "Tenants/GetSingle?id=" + id.ToString();
        }

        public static string UserViewsGetByFilters()
        {
            return "UserViews/GetByFilters";
        }

        public static string ContactViewsGetByFilters()
        {
            return "ContactViews/GetByFilters";
        }

        public static string ContactsGetSingle(string id)
        {
            return "Contacts/GetSingle?id=" + id;
        }

        public static string AddressViewsGetByFilters()
        {
            return "AddressViews/GetByFilters";
        }

        public static string AddressesGetSingle(string id)
        {
            return "Addresses/GetSingle?id=" + id;
        }

        public static string ShipmentViewsGetByFilters()
        {
            return "ShipmentViews/GetByFilters";
        }

        public static string ShipmentGetSingle(string id)
        {
            return "Shipment/GetSingle?id=" + id;
        }

        public static string Shipment()
        {
            return "Shipment";
        }

        public static string FTPDetailViewsGetByFilters()
        {
            return "FTPDetailViews/GetByFilters";
        }

        public static string FTPDetailsGetSingle(string id)
        {
            return "FTPDetails/GetSingle?id=" + id;
        }

        public static string APInvoiceViewsGetByFilters()
        {
            return "APInvoiceViews/GetByFilters";
        }

        public static string APInvoicesGetSingle(string id)
        {
            return "APInvoices/GetSingle?id=" + id;
        }

        public static string APPaymentViewsGetByFilters()
        {
            return "APPaymentViews/GetByFilters";
        }

        public static string APPaymentsGetSingle(string id)
        {
            return "APPayments/GetSingle?id=" + id;
        }

        public static string ARInvoiceViewsGetByFilters()
        {
            return "ARInvoiceViews/GetByFilters";
        }

        public static string ARInvoicesGetSingle(string id)
        {
            return "ARInvoices/GetSingle?id=" + id;
        }

        public static string ARPaymentViewsGetByFilters()
        {
            return "ARPaymentViews/GetByFilters";
        }

        public static string ARPaymentsGetSingle(string id)
        {
            return "ARPayments/GetSingle?id=" + id;
        }
    }
}

File: Logitude/AngularModules/CypressTesting/FullAccounting/cypress/constants/URLs.ts
export class URLs {
    public static readonly InvoicesGetSingle = '**/cardviews/getsingle/?**';
    public static readonly APInvoices = '**/apinvoices';
    public static readonly ARInvoices = '**/arinvoices';
    public static readonly ARInvoicesGetSingle = '**/arinvoices/getsingle?**';
    public static readonly APInvoiceViewsGetByFilters = '**/cardviews/getbyfilters?**';
    public static readonly ARInvoiceViewsGetByFilters = '**/arinvoiceviews/getbyfilters?**';
    public static readonly APPayments = '**/appayments';
    public static readonly ARPayments = '**/arpayments';
    public static readonly GLAccounts = '**/glaccounts';
    public static readonly Bankaccounts = '**/bankaccounts';
    public static readonly PartnersDomain = '**/PartnersDomain';
    public static readonly CustomerGetSingle = '**/customers/getsingle?**';
    public static readonly GLAccountGetSingle = '**/fullaccountingsettingviews/getsingle/?**';
    public static readonly ChartOfAccounts = '**/chartofaccounts';
    public static readonly ChartOfAccountsGetSingle = "**/chartofaccounts/getsingle?**";

    public static GetFilterSearch(filterBy: string) {
        return '**/getbyfilters?**' + filterBy + '**'
    }

    public static GetQuickSearch(filterBy: string) {
        return '**/GetQuickSearch?**' + filterBy + '**'
    }
}